### PR TITLE
update the documentation for jwt authentication.

### PR DESCRIPTION
**Pull request by yangminzhu**

### CODE REVIEW

**Comment by yangminzhu** [comment]
@geeknoid Could you review this? I think the circle CI build fail is not related to this PR. Thank you.

File: content/docs/tasks/security/authn-policy/index.md [hunk 1]
```diff
@@ -609,6 +609,11 @@ End-user authentication can be enabled or disabled based on request path. This i
 disable authentication for some paths, for example, the path used for health check or status report.
 You can also specify different JWT requirements on different paths.
 
+{{< warning >}}
+The End-user authentication with per-path requirements is an experimental feature in version 1.1 and
```
Comment (geeknoid) [inline content/docs/tasks/security/authn-policy/index.md]: End -> end
version 1.1 -> Istio 1.1

Comment (yangminzhu) [reply content/docs/tasks/security/authn-policy/index.md]: Done.

**Comment by geeknoid** [comment]
Rebasing and repushing should fix that CI failure.

**Comment by yangminzhu** [comment]
@geeknoid  all tests pass now.

**Pull request merged**